api-docs/release-notes/releases/cdbv3.10.0.rst

```diff
@@ -0,0 +1,43 @@
+.. version-3.10.0-release-notes:
+
+v3.10.0, October 27, 2020
+-------------------------
+
+What's new
+~~~~~~~~~~
+
+-  Upgrading a single instance will put both the instance and it's upgraded
+   replica into a VERIFY_UPGRADE state.
+
+   **Note:** Both the instance and upgraded replica will remain in this state
```

Comment:
**Note:** Both the instance and upgraded replica remain in this state   [removed will]